### ISSUE TITLE
Rename cas-server-webapp to cas?

### ISSUE THREAD

**Issue by severo** [georchestra/georchestra#2370]
I know that renaming a webapp is a good way to make the sysadmins cry, but... wouldn't it be simpler for the next users (we already use the name cas in the datadir repository)?

**Comment by landryb** [comment]
Where would the renaming be ? right now the build infra produces `cas-{template}.war` or `georchestra-cas*.deb` under `cas-server-webapp/target` .. so unless i'm missing smth, to me that's only renaming the toplevel dir in the source tree, and fixing the few occurences in the Makefile, some doc & pom.xml files ?

**Comment by severo** [comment]
Exactly, it's not a big change, but it will imply modifying deployment scripts, maybe supervision configuration, for those who use that... So, I'm wondering if it's a good idea or not.

**Comment by fvanderbiest** [comment]
I'm not against it. 
But I'd rather not do it for 18.12.
Thanks

**Comment by severo** [comment]
See https://github.com/georchestra/georchestra/issues/2370
